app/controllers/importer_controller.rb

```diff
@@ -731,13 +734,33 @@ def version_id_for_name!(project, name, add_versions)
     @version_id_by_name[name]
   end
 
-  def process_multivalue_custom_field(project, add_versions, issue, custom_field, csv_val)
+  def enumeration_id_for_name!(custom_field, name, add_enumerations)
+    unless @enumeration_id_by_name.key?(name)
+      enumeration = custom_field.enumerations.find_by(name: name).try!(:id)
+      if enumeration.nil?
+        if name.try! { |n| n.size > 1 } && add_enumerations
+          max_position = custom_field.enumerations.pluck(:position).max || 0
+          enumeration = CustomFieldEnumeration.create(custom_field: custom_field, name: name, position: max_position + 1).id
```

Comment:
> `position: max_position + 1`

Is this attribute needed? `CustomFieldEnumeration#set_position` sets position before creation by default.

Reply:
Thank you! I didn't know that. I'll fix it.

Reply:
Fixed https://github.com/agileware-jp/redmine_importer/pull/5/commits/a6ac56038aafa4a29e7f20c99b5e978e8cd5b243 💪